.changeset/ready-owls-heal.md

```diff
@@ -0,0 +1,94 @@
+"@fluidframework/dds-interceptions": major
+"@fluidframework/debugger": major
+"@fluid-experimental/devtools": major
+"@fluid-experimental/devtools-core": major
+"@fluidframework/driver-base": major
+"@fluidframework/driver-definitions": major
+"@fluidframework/driver-utils": major
+"@fluidframework/driver-web-cache": major
+"@fluid-tools/fetch-tool": major
+"@fluidframework/file-driver": major
+"fluid-framework": major
+"@fluidframework/fluid-runner": major
+"@fluidframework/fluid-static": major
+"@fluid-tools/fluidapp-odsp-urlresolver": major
+"@fluidframework/ink": major
+"@fluid-experimental/last-edited": major
+"@fluidframework/local-driver": major
+"@fluidframework/location-redirection-utils": major
+"@fluidframework/map": major
+"@fluidframework/matrix": major
+"@fluidframework/merge-tree": major
+"@fluidframework/mocha-test-setup": major
+"@fluidframework/odsp-doclib-utils": major
+"@fluidframework/odsp-driver": major
+"@fluidframework/odsp-driver-definitions": major
+"@fluidframework/odsp-urlresolver": major
+"@fluid-experimental/oldest-client-observer": major
+"@fluidframework/ordered-collection": major
+"@fluid-experimental/ot": major
+"@fluid-experimental/pact-map": major
+"@fluid-experimental/property-binder": major
+"@fluid-experimental/property-changeset": major
+"@fluid-experimental/property-common": major
+"@fluid-experimental/property-dds": major
+"@fluid-experimental/property-inspector-table": major
+"@fluid-experimental/property-properties": major
+"@fluid-experimental/property-proxy": major
+"@fluid-experimental/property-shared-tree-interop": major
+"@fluid-experimental/react-inputs": major
+"@fluidframework/register-collection": major
+"@fluidframework/replay-driver": major
+"@fluidframework/request-handler": major
+"@fluidframework/routerlicious-driver": major
+"@fluidframework/routerlicious-urlresolver": major
+"@fluidframework/runtime-definitions": major
+"@fluidframework/runtime-utils": major
+"@fluidframework/sequence": major
+"@fluid-experimental/sequence-deprecated": major
+"@fluidframework/shared-object-base": major
+"@fluidframework/shared-summary-block": major
+"@fluid-experimental/sharejs-json1": major
+"@fluidframework/synthesize": major
+"@fluidframework/task-manager": major
+"@fluidframework/telemetry-utils": major
+"@fluidframework/test-driver-definitions": major
+"@fluidframework/test-runtime-utils": major
+"@fluidframework/test-utils": major
+"@fluidframework/tinylicious-client": major
+"@fluidframework/tinylicious-driver": major
+"@fluidframework/tool-utils": major
+"@fluid-experimental/tree": major
+"@fluid-experimental/tree-react-api": major
+"@fluid-experimental/tree2": major
+"@fluidframework/undo-redo": major
+"@fluidframework/view-adapters": major
+"@fluidframework/view-interfaces": major
+"@fluid-tools/webpack-fluid-loader": major
+---
+
+Minimum TypeScript version now 5.1.6
+
+The minimum supported TypeScript version for Fluid 2.0 clients is now 5.1.6.
```

Comment:
I checked that at least our biggest internal partners are using a TS version greater than this, so hopefully this won't be an issue.